common/lib/api-client/models.js

```diff
@@ -237,6 +237,11 @@ module.exports = {
       defaultInclude: ['suppliers'],
     },
   },
+  locations_free_spaces: {
```

Comment:
As far as I can tell, adding this model made no difference - however it does seem safer to tie down our api-client to using known endpoints